commercetools-models/src/test/java/io/sphere/sdk/shippingmethods/commands/ShippingMethodDeleteCommandIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package io.sphere.sdk.shippingmethods.commands;
+
+import io.sphere.sdk.shippingmethods.ShippingMethod;
+import io.sphere.sdk.shippingmethods.commands.updateactions.SetKey;
+import io.sphere.sdk.shippingmethods.queries.ShippingMethodQuery;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import static io.sphere.sdk.shippingmethods.ShippingMethodFixtures.withUpdateableShippingMethod;
+import static io.sphere.sdk.test.SphereTestUtils.randomKey;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ShippingMethodDeleteCommandIntegrationTest extends IntegrationTest {
```

Comment:
The current delete command, has a delete `of(versioned)`, but there is not a delete `ofId(...)`. Do we need to implement the `ofId(...)` delete?, because I think it would be necessary to modify the generator for delete commands